eduvpn/actions/activate.py

```diff
@@ -17,12 +17,15 @@
 from eduvpn.exceptions import EduvpnAuthException, EduvpnException
 from eduvpn.crypto import common_name_from_cert
 from eduvpn.brand import get_brand
+from eduvpn.metadata import Metadata
+from gi.repository import Gtk
+from typing import Any, Union, List
 
 logger = logging.getLogger(__name__)
 
 
 # ui thread
-def activate_connection(meta, builder, verifier, lets_connect):
+def activate_connection(meta, builder, verifier, lets_connect):  # type: (Metadata, Gtk.builder, str, bool) -> None
```

Comment:
volgens mij wordt erg gecheckt of regels niet langer zijn dan 120 chars, en deze lijkt langer. In dat geval moet de type comment op een nieuwe regel.

Reply:
Deze is 115 characters anders was deze ook naar voren gekomen bij de travis tests.